app/test/page.tsx

```
@@ -0,0 +1,12 @@
+type Props = {};
+
+//  todo: align the carousel items height, use this page for testing
+export default function Page({}: Props) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)